src/Microsoft.Identity.Web.TokenAcquisition/ClaimConstants.cs

```diff
@@ -99,5 +99,15 @@ public static class ClaimConstants
         /// Name Identifier ID claim: "http://schemas.xmlsoap.org/ws/2005/05/identity/claims/nameidentifier".
         /// </summary>
         public const string NameIdentifierId = "http://schemas.xmlsoap.org/ws/2005/05/identity/claims/nameidentifier";
+
+        /// <summary>
+        /// Username claims for ROPC flow.
+        /// </summary>
+        public const string Username = "xms-username";
+
+        /// <summary>
+        /// Password claims for ROPC flow.
+        /// </summary>
+        public const string Password = "xms-password";
```

Comment:
You'll need to add a suppression line in credscan files in the builds

Reply:
Can you point me to the file?

Reply:
Do you mean the PublicApi.Unshipped.txt files?

Reply:
Added